FamilyTree/po/it-local.po

```diff
@@ -19,291 +19,292 @@ msgstr ""
 
 #: FamilyTree/FamilyTree.gpr.py:3
 msgid "Family Tree"
-msgstr ""
+msgstr "Albero Genealogico"
 
 #: FamilyTree/FamilyTree.gpr.py:4
 msgid "Produces a graphical family tree."
-msgstr ""
+msgstr "Genera un albero genealogico grafico"
 
 #: FamilyTree/FamilyTree.py:102
 #, python-format
 msgid "Family Tree for %s"
-msgstr ""
+msgstr "Albero Genealogico per %s"
 
 #: FamilyTree/FamilyTree.py:668 FamilyTree/FamilyTree.py:673
 msgid "Unknown"
-msgstr ""
+msgstr "Sconosciuto"
 
 #: FamilyTree/FamilyTree.py:675
 #, python-format
 msgid "%(father)s and %(mother)s"
-msgstr ""
+msgstr "%(padre)s e %(madre)s"
 
 #: FamilyTree/FamilyTree.py:682
 msgid "Anonymous"
-msgstr ""
+msgstr "Anonimo"
 
 #: FamilyTree/FamilyTree.py:905
 msgid "born"
-msgstr ""
+msgstr "nato"
 
 #: FamilyTree/FamilyTree.py:907
 msgid "baptised"
-msgstr ""
+msgstr "battezzato"
 
 #: FamilyTree/FamilyTree.py:909
 msgid "died"
-msgstr ""
+msgstr "morto"
 
 #: FamilyTree/FamilyTree.py:911
 msgid "buried"
-msgstr ""
+msgstr "seppellito"
 
 #: FamilyTree/FamilyTree.py:913
 msgid "cremated"
-msgstr ""
+msgstr "cremato"
 
 #: FamilyTree/FamilyTree.py:915
 msgid "married"
-msgstr ""
+msgstr "sposato"
 
 #: FamilyTree/FamilyTree.py:917
 msgid "divorced"
-msgstr ""
+msgstr "divorziato"
 
 #: FamilyTree/FamilyTree.py:919
 msgid "resident"
-msgstr ""
+msgstr "residente"
 
 #: FamilyTree/FamilyTree.py:926
 #, python-format
 msgid "on %(ymd_date)s"
-msgstr ""
+msgstr "il %(ymd_date)s"
 
 #: FamilyTree/FamilyTree.py:928
 #, python-format
 msgid "in %(ym_date)s"
-msgstr ""
+msgstr "il %(ym_date)s"
 
 #: FamilyTree/FamilyTree.py:930
 #, python-format
 msgid "in %(y_date)s"
-msgstr ""
+msgstr "il %(y_date)s"
 
 #: FamilyTree/FamilyTree.py:934
 #, python-format
 msgid "on %(placeholder)s"
-msgstr ""
+msgstr "in %(placeholder)s"
 
 #: FamilyTree/FamilyTree.py:937
 #, python-format
 msgid "on %(placeholder)s (%(partial)s)"
-msgstr ""
+msgstr "in %(placeholder)s (%(partial)s)"
 
 #: FamilyTree/FamilyTree.py:954
 #, python-format
 msgid "in %(place)s"
-msgstr ""
+msgstr "in %(place)s"
 
 #: FamilyTree/FamilyTree.py:965
 #, python-format
 msgid "(%(description)s)"
-msgstr ""
+msgstr "(%(description)s)"
 
 #: FamilyTree/FamilyTree.py:1043
 msgid "Tree Options"
-msgstr ""
+msgstr "Opzioni per l'albero"
 
 #: FamilyTree/FamilyTree.py:1045
 msgid "Center Family"
-msgstr ""
+msgstr "Famiglia Principale"
```

Comment:
No camel case, that's only an English way of writing (and not the only one): "Famiglia principale"